test/test_jit.py

```diff
@@ -2070,6 +2070,21 @@ def fn(x):
         warns = [str(w.message) for w in warns]
         self.assertEqual(len(warns), 0)
 
+    def test_torch_load_error(self):
+        class J(torch.jit.ScriptModule):
+            def __init__(self):
+                super(J, self).__init__()
+
+            @torch.jit.script_method
+            def forward(self, input):
+                return input + 100
+
+        j = J()
+        with tempfile.NamedTemporaryFile() as f:
```

Comment:
We can not reference these files through the name on Windows. See how we addressed them [here](https://github.com/pytorch/pytorch/pull/4065/files#diff-9996665f82f52030836eb8657057cfadR4244).